client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -92,7 +92,7 @@ function EditCardDetails( props ) {
 						configurationArgs={ { needs_intent: true } }
 						fetchStripeConfiguration={ getStripeConfiguration }
 					>
-						<CreditCardForm
+						<PaymentMethodForm
```

Comment:
Sorry about the mixup. There's a lot going here. You are both right that I should leave things as is and use the flag to display the `ChangePaymentMethod` component instead of this one.